### ISSUE TITLE
"Matrix tester" not working for rotary encoder

### ISSUE THREAD

**Issue by nwhirschfeld** [vial-kb/vial-gui#78]
At least for me, Vial GUI is able to change the mapping of a rotary encoder but not showing rotation in the "Matrix tester". For the button in my test setup, the rotary encoder works great.
This is misleading, because it makes the user think that the encoder is not working correctly.
I tested everything with a custom handwired (for now on a breadboard). Vial GUI version 0.4.2 under fedora Linux and vial-qmk commit bf3988680a92849ccddc07079b99a0b9f8573891.

**Comment by xyzz** [comment]
Yes, since this is originally VIA functionality, it does not support Vial encoders. We should get it fixed however.
